lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
OK it seems to be fine after creating the symlink to `/Applications/GAP/gap4r7/pkg/`.  Here is the output.

    make manuals
    ( echo 'SaveWorkspace( "doc/wsp.g" );' | ./bin/gap-default64.sh -b -m 100m -o 750m -q -x 80 -r  )
    true
    ( cd doc/ref; echo 'Read( "makedocrel.g" );' | \
          ../.././bin/gap-default64.sh -b -m 100m -o 750m -A -q -x 80 -r  -L ../wsp.g > make_manuals.out )
    ( cd doc/tut; echo 'Read( "makedocrel.g" );' | \
          ../.././bin/gap-default64.sh -b -m 100m -o 750m -A -q -x 80 -r  -L ../wsp.g > make_manuals.out )
    ( cd doc/changes; echo 'Read( "makedocrel.g" );' | \
          ../.././bin/gap-default64.sh -b -m 100m -o 750m -A -q -x 80 -r  -L ../wsp.g > make_manuals.out )          
    ( cd doc/ref; echo 'Read( "makedocrel.g" );' | \
          ../.././bin/gap-default64.sh -b -m 100m -o 750m -A -q -x 80 -r  -L ../wsp.g > make_manuals.out )
    ( cd doc/tut; echo 'Read( "makedocrel.g" );' | \
          ../.././bin/gap-default64.sh -b -m 100m -o 750m -A -q -x 80 -r  -L ../wsp.g > make_manuals.out )
    ( cd doc/changes; echo 'Read( "makedocrel.g" );' | \
          ../.././bin/gap-default64.sh -b -m 100m -o 750m -A -q -x 80 -r  -L ../wsp.g > make_manuals.out )           
    ( rm doc/wsp.g )